docs/source/getting_started/woodwork_types.ipynb

```diff
@@ -0,0 +1,385 @@
+   "source": [
+    "Featuretools will perform type inference to assign logical types to the data in EntitySets if none are provided, but it is also possible to specify which logical types should be set for any column (provided that the data in that column is compatible with the logical type).\n",
+    "\n",
+    "To learn more about how logical types are used in EntitySets, see the [Creating EntitySets](using_entitysets.ipynb) guide.\n",
+    "\n",
+    "To learn more about setting logical types directly on a DataFrame, see the Woodwork guide on [working with Logical Types](https://woodwork.alteryx.com/en/stable/guides/working_with_types_and_tags.html#Working-with-Logical-Types). \n",
+    "\n",
+    "## Semantic Tags\n",
+    "Semantic tags provide additional information to columns about the meaning or potential uses of data. Columns can have many or no semantic tags. Some tags are added by Woodwork, some are added by Featuretools, and users can add additional tags as they see fit.\n",
+    "\n",
+    "To learn more about setting semantic tags directly on a DataFrame, see the Woodwork guide on [working with Semantic Tags](https://woodwork.alteryx.com/en/stable/guides/working_with_types_and_tags.html#Working-with-Semantic-Tags). \n",
+    "\n",
+    "### Woodwork Standard Tags\n",
+    "\n",
+    "Woodwork will add certain semantic tags to columns at initialization. These can be standard tags that may be associated with different sets of logical types, index tags, or even tags that have some additional meaning in Woodwork.\n",
+    "\n",
+    "To get a list of the standard, index, and time index tags, you can use the `list_semantic_tags` function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "11f25bd9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ft.list_semantic_tags()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29222810",
+   "metadata": {},
+   "source": [
+    "Above we see the standard tags that are defined within Woodwork. These tags inform how Featuretools is able to interpret data, but in order to generate the full set of features, Featuretools has defined a few tags of its own.\n",
+    "\n",
+    "### Semantic Tags in Featuretools\n",
+    "\n",
+    "Just like Woodwork specifies semantic tags internally, Featuretools also has certain semantic tags that have a specific meaning when they are present on a column.\n",
+    "\n",
+    "- `'last_time_index'` - added by Featuretools to the last time index column of a DataFrame. Indicates that this column has been created by Featuretools.\n",
+    "- `'foreign_key'` - used to indicate that this column is the child column of a relationship, meaning that this column is related to a corresponding index column of another dataframe in the EntitySet.\n",
+    "- `'date_of_birth'` - indicates that a column can be used for Primitive operations that require a date of birth. Must be added at EntitySet creation to be present.\n",
```

Comment:
`date_of_birth` also shows up in the `ft.list_semantic_tags` call previously

Reply:
Yeah... We were just having a conversation about this on https://github.com/alteryx/woodwork/pull/1094 . Maybe the presence in `list_semantic_tags` and the explanation in that guide in Woodwork is enough, but I do still feel like the description here adds a bit of featuretools-specific information that makes it still useful. 

Happy to remove and find another way to get this information across (maybe putting it below the `list_semantic_tags` output?) if it seems redundant this way.

Reply:
What if we removed it from this list but in the previous paragraph, we add a sentence / thought giving an example of how Featuretools uses semantics tags, and that example happens to be date of birth

Reply:
Doing this makes sense. Also, given @thehomebrewnerd's comments about the `date_of_birth` type semantic tags in Woodwork, I reworked the semantic tags section a bit to be more in line with that way of thinking about these types of tags.